api/turing/cluster/servicebuilder/service_builder.go

```diff
@@ -287,11 +289,13 @@ func (sb *clusterSvcBuilder) NewSecret(
 	routerServiceAccountKey string,
 	enricherServiceAccountKey string,
 	ensemblerServiceAccountKey string,
+	expEngineServiceAccountKey string,
 ) *cluster.Secret {
 	data := map[string]string{
 		secretKeyNameRouter:    routerServiceAccountKey,
 		secretKeyNameEnricher:  enricherServiceAccountKey,
 		secretKeyNameEnsembler: ensemblerServiceAccountKey,
+		secretKeyNameExpEngine: expEngineServiceAccountKey,
```

Comment:
This is good. But we also need to mount this key to the router, right (like [this](https://github.com/caraml-dev/turing/blob/8002d13de90f81efbeb545c02c5504bd35e3c132/api/turing/cluster/servicebuilder/router.go#L328))? Will that be done in a different PR?

Reply:
Oh that's true, I must've missed that out; I'll add this to this PR.